package.json

```diff
@@ -77,7 +77,7 @@
     "test:unit": "jest"
   },
   "dependencies": {
-    "3box": "1.16.1",
+    "3box": "1.17.1",
```

Comment:
is this related to this pr ?

Reply:
this is from the Dao follow/unfollow PR. it will go away when dev is merged in again